app/views/measures/_measure.html.erb

```diff
@@ -25,13 +25,14 @@
           declarable_id: declarable.short_code %>
       </span>
     <% end %>
-
```

Comment:
Why compress these blocks together? Seems like it reduces rather then increases readability?